README.md

```diff
@@ -1,59 +1,127 @@
 
-    <div id="cronofy-calendar"></div>
-    <script src="https://s3.eu-west-2.amazonaws.com/component-demo/CronofyComponents.v0.1.0.js"></script>
-    <script>
-        CronofyComponents.Agenda({token: "YOUR_TOKEN",target: "cronofy-calendar"});
-    </script>
+### Authenticated mode vs. Demo mode
 
-### Demo mode
+CronofyComponents all require two things to run: an authentication token, and a target DOM element (to load the component into). The "token" value should be the authorization token from the API as a string, and the target DOM element is indicated by a unique `ID` string.
 
-For demo purposes, it is possible to bypass the authentication step. If the `demo` option is set to `true`, the component will not make any API calls and will display mock content.
+Using the above `Agenda()` example, CronofyComponents will look for a DOM element with the ID `cronofy-calendar`, and inject the CronofyComponents component inside that element.
+
+A simple integration into an `index.html` file would look something like this:
 
     <div id="cronofy-calendar"></div>
     <script src="https://s3.eu-west-2.amazonaws.com/component-demo/CronofyComponents.v0.1.0.js"></script>
     <script>
         CronofyComponents.Agenda({
-            target: "cronofy-calendar",
-            demo: true
+            token: "YOUR_TOKEN",
+            target: "cronofy-calendar"
         });
     </script>
 
-## Theming
-
-CronofyComponents will automatically inherit the typeface used in your page. You can assign a custom color palette for the calendars by passing in an array of HEX values to the `colors.calendars` option. For example:
+For demo purposes, it is possible to bypass the authentication step. If the `demo` option is set to `true`, the component will not make any API calls and will display mock content.
 
     <div id="cronofy-calendar"></div>
     <script src="https://s3.eu-west-2.amazonaws.com/component-demo/CronofyComponents.v0.1.0.js"></script>
     <script>
         CronofyComponents.Agenda({
-            token: "YOUR_TOKEN",
             target: "cronofy-calendar",
-            colors: { calendars: ['#0074d9','#ffdc00'] }
+            demo: true
         });
     </script>
 
+### Global options
+
+These apply to all components.
+
+* `token` [required]: auth token for API connection. *Not required if the component is activated in demo mode.*
+* `demo`: boolean to activate demo-mode. Defaults to `false`. If `demo` is set to `true` the component will return mock data (and not make any API calls). 
+* `target` [required]: ID of mounting element (e.g. 'cronofy-slot-picker').
+* `api_domain` [optional]: Override the default Default value is `"http://api.cronofy.com"`.
+* `styles` [optional]: an object containing you custom style values See [Custom Styles](#custom-styles) for details of valid options.
+
+
+### Agenda options
+
+The Agenda component 
```

Comment:
Not sure what happened here 😛 - should read:

"The Agenda component can be assigned a custom color palette for the calendars by passing in an array of HEX values to the `colors.calendars` option. For example:	`colors: { calendars: ['#0074d9','#ffdc00'] }`"